src/vs/workbench/contrib/chat/browser/actions/chatActions.ts

```diff
@@ -561,7 +561,7 @@ export class ChatCommandCenterRendering extends Disposable implements IWorkbench
 
 		const contextKeySet = new Set([ChatContextKeys.Setup.signedOut.key]);
 
-		actionViewItemService.register(MenuId.CommandCenter, MenuId.ChatCommandCenter, (action, options) => {
+		this._store.add(actionViewItemService.register(MenuId.CommandCenter, MenuId.ChatCommandCenter, (action, options) => {
```

Comment:
I think this one causes the 🤖 icon to come and go on window reloads, so I had it explicitly not to dispose.

Reply:
Yeah true, will see if it still flickers, good idea!